zokrates_core/src/proof_system/ark/mod.rs

```diff
@@ -148,35 +141,20 @@ impl<T: Field + ArkFieldExtensions, I: IntoIterator<Item = Statement<T>>> ProgIt
     }
 }
 
-impl<T: Field + ArkFieldExtensions, I: IntoIterator<Item = Statement<T>>> Computation<T, I> {
-    pub fn prove(self, params: &ProvingKey<T::ArkEngine>) -> Proof<T::ArkEngine> {
-        let rng = &mut rand_0_7::rngs::StdRng::from_entropy();
-
-        let public_inputs = self.public_inputs_values();
-
-        let proof = create_random_proof(self, params, rng).unwrap();
-
-        let pvk = prepare_verifying_key(&params.vk);
-
-        assert!(verify_proof(&pvk, &proof, &public_inputs).unwrap());
-
-        proof
-    }
+pub fn get_random_seed() -> Result<[u8; 32], getrandom::Error> {
```

Comment:
Can you explain why this change is required in this PR? Did the ark API change? If so, what did it use under the hood before for randomness?

Reply:
We used `from_entropy()` function from `rand 0.7` which does not work well with wasm. It compiles fine but you get a runtime error `target not supported`. Newest versions of rand `>0.8` use this `getrandom` crate under the hood.

Reply:
Actually, there is a feature on `rand 0.7` we can enable regarding wasm-bindgen, so I will revert these changes!